src/main/java/com/qunite/api/QuniteApiApplication.java

```diff
@@ -10,4 +17,8 @@ public static void main(String[] args) {
         SpringApplication.run(QuniteApiApplication.class, args);
     }
 
+
+    private void DoNothing(){
+
+    }
```

Comment:
⚠️ **[reviewdog]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 4, expected level should be 2.